### PR TITLE
En 5671 improve code coverage

### PR DESCRIPTION
**Pull request by miiu96**

### CODE REVIEW

File: statusHandler/statusMetricsProvider.go [hunk 1]
```diff
@@ -18,10 +18,7 @@ func NewStatusMetrics() *statusMetrics {
 
 // IsInterfaceNil returns true if there is no value under the interface
 func (nd *statusMetrics) IsInterfaceNil() bool {
```
Comment (SebastianMarian) [inline statusHandler/statusMetricsProvider.go]: sm instead nd as pointer receiver name?

File: statusHandler/statusMetricsProvider_test.go [hunk 2]
```diff
@@ -95,3 +95,17 @@ func TestStatusMetricsProvider_SetStringValue(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, value, retMap[key])
 }
+
+func TestStatusMetricsProvider_AddUint64Value(t *testing.T) {
+	t.Parallel()
+
+	ndh := statusHandler.NewStatusMetrics()
```
Comment (SebastianMarian) [inline statusHandler/statusMetricsProvider_test.go]: sm instead ndh in whole this file?

File: core/statistics/softwareVersion/stableTagProvider.go [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+	defer func() {
+		err = resp.Body.Close()
+		if err != nil {
+			log.Debug(err.Error())
+		}
+	}()
+
+	buf := new(bytes.Buffer)
+	_, err = buf.ReadFrom(resp.Body)
```
[the remark this answers is not in the document]
Comment (raduchis) [reply core/statistics/softwareVersion/stableTagProvider.go]: :+1:

File: core/statistics/softwareVersion/softwareVersion_test.go [hunk 4]
```diff
@@ -11,18 +13,95 @@ import (
...
+	softwareChecker.StartCheckSoftwareVersion()
+
+	select {
+	case <-fetchChan:
+	case <-time.After(1 * time.Second):
+		assert.Fail(t, "fetch was not called")
+	}
+
+	select {
+	case <-time.After(10 * time.Millisecond):
```
[the remark this answers is not in the document]
Comment (miiu96) [reply core/statistics/softwareVersion/softwareVersion_test.go]: you right maybe 10 millisecond is not enough changed in 100 ms

File: core/statistics/softwareVersion/stableTagProvider.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (miiu96) [reply core/statistics/softwareVersion/stableTagProvider.go]: done.

File: statusHandler/persister/persistentHandler.go [hunk 5]
```diff
@@ -131,6 +135,8 @@ func (psh *PersistentStatusHandler) SetUInt64Value(key string, value uint64) {
 		return
 	}
 
+	psh.mutex.Lock()
```
[the remark this answers is not in the document]
Comment (miiu96) [reply statusHandler/persister/persistentHandler.go]: done.

File: statusHandler/statusMetricsProvider.go (hunk 1 above)
Comment (miiu96) [reply statusHandler/statusMetricsProvider.go]: done.

File: statusHandler/statusMetricsProvider_test.go (hunk 2 above)
Comment (miiu96) [reply statusHandler/statusMetricsProvider_test.go]: done.

**Pull request merged**